go/stats/export.go

```diff
@@ -183,8 +199,10 @@ func publish(name string, v expvar.Var) {
 // to be pushed to it. It's used to support push-based metrics backends, as expvar
 // by default only supports pull-based ones.
 type PushBackend interface {
-	// PushAll pushes all stats from expvar to the backend
+	// PushAll pushes all stats from expvar to the backend.
 	PushAll() error
+	// PushOne pushes a single stat from expvar to the backend.
+	PushOne(name string, v expvar.Var) error
```

Comment:
This may be a breaking change for out-of-tree push backends. Should I add a note in the changelog?